src/interfaces/note.ts

```diff
@@ -0,0 +1,7 @@
+export class Note{
```

Comment:
ten plik powinien być w folderze note

Reply:
widzę też, że ten plik jest używany jak DTO do body w POST, moznaby się zastanowić nad zmianą nazwy na NoteDTO.

Konieczne jest tutaj dodanie walidacji

Reply:
Poprawiłem. Walidację  odłożyłem z powodu pogoni za nowym :)

Reply:
Jeśli chodzi o formotowanie  w NoteDTO to przeprowadziłem mały eksperyment No nie wiem zawsze można zmienić ... ale teraz wydaje mi się że to co valiuje jest wystające :) a to czym validuje jest jakby w drugiej lini